test/index_tests.js

```diff
@@ -62,7 +62,9 @@ describe('index.js', () => {
 
   beforeAll(done => {
     const app = express();
-    generator.init(app, {});
+
+    //Default args supplied to init. Only tag is modified
+    generator.init(app, {}, undefined, 1000*10, 'api-docs', [], ['hello']);
```

Comment:
Consult @mpashkovskiy to make sure this does not affect other tests in unexpected ways

Reply:
It doesn't because they are the default values. But anyway I'm removing and creating a new dataset.

Reply:
Moved tags test to an separated generator. Back to `generator.init(app, {})` on this one.